Marlin/src/lcd/language/language_de.h

```diff
@@ -200,9 +200,9 @@
 #define MSG_MOVE_E                          _UxGT("Bewege Extruder")
 #define MSG_HOTEND_TOO_COLD                 _UxGT("Hotend zu kalt")
 #define MSG_MOVE_0025MM                     _UxGT(" 0,025 mm")
-#define MSG_MOVE_01MM                       _UxGT(" 0,1 mm")
-#define MSG_MOVE_1MM                        _UxGT(" 1,0 mm")
-#define MSG_MOVE_10MM                       _UxGT("10,0 mm")
+#define MSG_MOVE_01MM                       _UxGT(" 0,100 mm")
+#define MSG_MOVE_1MM                        _UxGT(" 1,000 mm")
+#define MSG_MOVE_10MM                       _UxGT("10,000 mm")
 #define MSG_SPEED                           _UxGT("Geschw.")
```

Comment:
Do you have more than 1000 steps/mm? This suggests an exactness there isn't. If you just want to align the "mm" replace the zeros with spaces.